tests/planedata/test_exclusion.py

```diff
@@ -24,10 +24,7 @@ def test_exclusion():
     assert pd.nHKLs == 2
     assert pd.getNhklRef() == 3
 
-    try:
-        # Exclude a range
-        pd.exclusions = [[0, 2]]
-        assert pd.nHKLs == 1
-        assert pd.getNhklRef() == 3
-    except NotImplementedError:
-        pass
```

Comment:
@donald-e-boyce was this supposed to intentionally raise a `NotImplementedError`?

Reply:
I wrote this test case.  It used to raise a NotImplementedError, but I implemented it so I removed the check in the test case

Reply:
Sounds good.